src/framework/global/tests/CMakeLists.txt

```diff
@@ -46,6 +46,12 @@ set(MODULE_TEST_SRC
     ${CMAKE_CURRENT_LIST_DIR}/version_tests.cpp
     ${CMAKE_CURRENT_LIST_DIR}/number_tests.cpp
     ${CMAKE_CURRENT_LIST_DIR}/ziprw_tests.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/msgpack_tests.cpp
 )
 
 include(SetupGTest)
+
+target_sources(muse_global_tests
```

Comment:
Looks like Qt Creator added something for you that you don't want...

Reply:
thank you, fixed